internal/declarative/v2/reconciler.go

```diff
@@ -24,6 +24,10 @@ var (
 	ErrObjectHasEmptyState                       = errors.New("object has an empty state")
 )
 
+const (
+	kymaSystem = "kyma-system"
```

Comment:
this one can be renamed to something like `namespaceNotBeRemoved` so that it's clear why we need to ignore it.